src/auth/auth.utils.ts

```diff
@@ -0,0 +1,36 @@
+import { AuthStrategy, authStrategyValues } from '~/auth/strategies/strategies.consts';
+
+/**
+ * transforms a raw auth strategy config into valid strategy values
+ *
+ * @throws if given invalid values
+ */
+export const parseAuthStrategyConfig = (rawStrategyConfig: string): AuthStrategy[] => {
+  const givenStrategies = rawStrategyConfig.split(',').map(strategy => strategy.trim());
+
+  const filteredStrategies = givenStrategies.filter(isStrategyKey);
+
+  if (filteredStrategies.length !== givenStrategies.length) {
+    throw new Error(
+      `Auth config error! "${givenStrategies}" contains an unrecognized option. Valid values are: "${authStrategyValues}"`
```

Comment:
Maybe a bit more user friendly way would be to remind that they should include the auth strategy in strategies.consts
e.g. "Did you forget to add the auth strategy to ~/auth/strategies/strategies.consts"